### PR TITLE
unsorted changes

### PR DESCRIPTION
**Pull request by Ladicek**

### CODE REVIEW

**Comment by Ladicek** [comment]
Probably easiest to review commit by commit.

**Review by sberyozkin — APPROVED** [review]

**Comment by sberyozkin** [comment]
Sorry, pressed too fast, give me few more mins please :-)

File: src/main/java/org/wildfly/swarm/plugin/process/ModuleRewriteConf.java [hunk 1]
```diff
@@ -191,6 +202,8 @@ private void load(Path file) throws IOException {
 
     private static final String FORCE_ARTIFACT_VERSION = "force-artifact-version:";
 
+    private static final String REPLACE_ARTIFACT = "replace-artifact:";
```
Comment (sberyozkin) [inline src/main/java/org/wildfly/swarm/plugin/process/ModuleRewriteConf.java:205]: When this option can be used ?

File: src/main/java/org/wildfly/swarm/plugin/utils/DescriptorUtils.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
+package org.wildfly.swarm.plugin.utils;
+
+import org.jboss.shrinkwrap.descriptor.api.Descriptor;
+import org.jboss.shrinkwrap.descriptor.spi.node.Node;
+import org.jboss.shrinkwrap.descriptor.spi.node.NodeDescriptor;
+
+public class DescriptorUtils {
```
Comment (sberyozkin) [inline src/main/java/org/wildfly/swarm/plugin/utils/DescriptorUtils.java:7]: This class is only used in `ModuleRewriteRules` but if you prefer to keep it standalone (may be other classes will refer to it as well) then it is fine as well

File: src/main/java/org/wildfly/swarm/plugin/process/ModuleRewriteConf.java (hunk 1 above)
Comment (Ladicek) [reply src/main/java/org/wildfly/swarm/plugin/process/ModuleRewriteConf.java:205]: RESTEasy changed one artifactId from `org.jboss.resteasy:resteasy-validator-provider-11:*` to `org.jboss.resteasy:resteasy-validator-provider:${version.resteasy}`

Comment (sberyozkin) [reply src/main/java/org/wildfly/swarm/plugin/process/ModuleRewriteConf.java:205]: I see, `replace-artifact` will be a useful feature

**Comment by sberyozkin** [comment]
Please go ahead with the merge, thanks

File: src/main/java/org/wildfly/swarm/plugin/utils/DescriptorUtils.java (hunk 2 above)
Comment (Ladicek) [reply src/main/java/org/wildfly/swarm/plugin/utils/DescriptorUtils.java:7]: Wanted to extract that code from the class, as it has a lot of inner classes.

**Pull request merged**